spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+describe Comment do
+  describe "table" do
+    it { should have_db_column(:rating).of_type(:integer) }
+    it { should have_db_column(:body).of_type(:string) }
+    it { should have_db_column(:created_at).of_type(:datetime) }
+    it { should have_db_column(:updated_at).of_type(:datetime) }
+  end
+
+  describe "associations" do
+    it { should belong_to(:user) }
+    it { should belong_to(:move) }
+  end
+
+  describe "validations" do
+    it { should have_valid(:rating).when (5) }
+    it { should validate_inclusion_of(:rating).in_range(1..5).
+                with_message(/ must be between 1 to 5/)}
```

Comment:
Expression at 19, 56 should be on its own line.<br>Use 2 (not 12) spaces for indenting an expression spanning multiple lines.